tpl/strings/strings.go

```diff
@@ -75,6 +78,11 @@ func (ns *Namespace) CountWords(s interface{}) (int, error) {
 		return 0, _errors.Wrap(err, "Failed to convert content to string")
 	}
 
+	isCJKLanguage := cjkRe.Match([]byte(ss))
```

Comment:
Use MatchString (which would save some memory allocation). Other than that, this looks great.

Reply:
Also: Could you change the commit message to say "tpl: Fix ..." or something (as this is a bug fix, we use the "fix" word to put them in the right category in the release notes). Also end the commit message with "Fixes #id".

Reply:
Done!